test/distributed/org/apache/cassandra/distributed/test/cql3/StatefulASTBase.java

```diff
@@ -144,7 +149,7 @@ protected static String nextKeyspace()
 
     protected void clusterConfig(IInstanceConfig config)
     {
-
+        config.set("repair.retries.max_attempts", Integer.MAX_VALUE);
```

Comment:
these are jvm-dtest clusters, retry non stop until it works... these tests don't do chaos testing, so its really the "happy path" case